### DISCUSSION TITLE
I am getting error to run simulation in debug mode

### DISCUSSION THREAD

**Discussion by j-iqu** [Unipisa/Simu5G#248] [Q&A] [answered]
Hi all,

any idea what this error could be? it looks there is a misconfiguration for running debug mode. I have not been able to run debug mode for simu5G, and now inet too.
![image](https://github.com/user-attachments/assets/506ef2d1-347b-4782-b3d3-b9172b0401ee)

**Comment by rhornig** [comment, marked as answer]
make sure you `source setenv` also from the INET root directory and then start the IDE from that shell.

**Reply by j-iqu** [reply]
Thank you! I tried, and it worked. I run OMNet++ from the INET root directory after typing the ```source setenv```
